web/src/main/webapp/resources/css/main.css

```diff
@@ -36,3 +36,48 @@
     border-top-left-radius: 0;
     border-top-right-radius: 0;
 }
+
+.match-form {
+    padding: 10px;
+    margin: 50px;
+    border-style: solid;
+    border-color: lightgrey;
+    border-width: 1px;
+    border-right: 0px;
+    border-left: 0px;
+}
+
+#match-form-legend {
+    margin-bottom: 40px;
+}
+
+.match-form-item {
+    margin: 5px;
+    margin-left: 0px;
+    padding: 5px;
+}
+
+.match-dropdown-items {
+    margin-left: -10px;
+    margin-top: 15px;
+    padding: 2px;
+    border-color: lightgrey;
+}
+
+.match-form-team {
+    width: 100%;
+}
+
+#form-buttons {
```

Comment:
ok, no popravde se se styly nechci nejak zatezovat, jen prebiram to, co bylo k navrhu sablony dodano a melo to svym zpusobem plnit svuj ucel...